api/src/main/java/ai/djl/training/listener/TrainingListener.java

```diff
@@ -97,6 +97,21 @@
             };
         }
 
+        /**
+         * A default {@link TrainingListener} set including batch output logging.
+         *
+         * @param frequency the frequency of epoch to print out
+         * @return the new set of listeners
+         */
+        static TrainingListener[] logging(int frequency) {
```

Comment:
Can we create a default, like 1 in here?

Reply:
The logging() listener set is still there with a frequency of 1.